### PR TITLE
New output directory structure

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/germline/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by nvnieuwk** [comment]
Don't merge before #213 has been merged

**Comment by nvnieuwk** [comment]
The new output directory now looks like this (for the test profile):

```
results/
├── Proband_12345
│   ├── NA24143_v1_9_0dev_2024_11_07
│   │   ├── NA24143.bed
│   │   ├── NA24143.haplotypecaller.bcftools_stats.txt
│   │   ├── NA24143.haplotypecaller.g.vcf.gz
│   │   └── NA24143.haplotypecaller.g.vcf.gz.tbi
│   ├── NA24149_v1_9_0dev_2024_11_07
│   │   ├── NA24149.bed
│   │   ├── NA24149.haplotypecaller.bcftools_stats.txt
│   │   ├── NA24149.haplotypecaller.g.vcf.gz
│   │   └── NA24149.haplotypecaller.g.vcf.gz.tbi
│   ├── NA24385_v1_9_0dev_2024_11_07
│   │   ├── NA24385.bed
│   │   ├── NA24385.haplotypecaller.bcftools_stats.txt
│   │   ├── NA24385.haplotypecaller.g.vcf.gz
│   │   └── NA24385.haplotypecaller.g.vcf.gz.tbi
│   ├── output_v1_9_0dev_2024_11_07
│   │   ├── automap
│   │   │   └── haplotypecaller
│   │   │       ├── sample1
│   │   │       │   ├── sample1.HomRegions.cmgg_bio.tsv
│   │   │       │   ├── sample1.HomRegions.pdf
│   │   │       │   ├── sample1.HomRegions.strict.cmgg_bio.tsv
│   │   │       │   └── sample1.HomRegions.tsv
│   │   │       ├── sample2
│   │   │       │   ├── sample2.HomRegions.cmgg_bio.tsv
│   │   │       │   ├── sample2.HomRegions.pdf
│   │   │       │   ├── sample2.HomRegions.strict.cmgg_bio.tsv
│   │   │       │   └── sample2.HomRegions.tsv
│   │   │       └── sample3
│   │   │           ├── sample3.HomRegions.cmgg_bio.tsv
│   │   │           ├── sample3.HomRegions.pdf
│   │   │           ├── sample3.HomRegions.strict.cmgg_bio.tsv
│   │   │           └── sample3.HomRegions.tsv
│   │   ├── Proband_12345.haplotypecaller.bed
│   │   ├── Proband_12345.haplotypecaller.db
│   │   ├── Proband_12345.haplotypecaller.ped
│   │   ├── Proband_12345.haplotypecaller.vcf.gz
│   │   └── Proband_12345.haplotypecaller.vcf.gz.tbi
│   └── qc_v1_9_0dev_2024_11_07
│       ├── Proband_12345.haplotypecaller.bcftools_stats.txt
│       └── Proband_12345.haplotypecaller.html
└── v1_9_0dev_2024_11_07
    ├── execution_report_2024-11-07_17-33-49.html
    ├── execution_timeline_2024-11-07_17-33-49.html
    ├── execution_trace_2024-11-07_17-33-49.html
    ├── multiqc_report.html
    ├── params_2024-11-07_17-34-02.json
    ├── pipeline_dag_2024-11-07_17-33-49.html
    ├── pipeline_software_mqc_versions.yml
    └── samplesheet.csv
```

File: nextflow.config
Comment (matthdsm) [inline nextflow.config:304]: waarom de revert?

**Review by matthdsm — APPROVED** [review]
LGTM, One small question

File: nextflow.config
Comment (nvnieuwk) [reply nextflow.config:304]: It's a version bump, not a revert :)

**Pull request merged**
